frontend/lobby/join/join.js

```diff
@@ -3,10 +3,11 @@
 let params = new URLSearchParams(document.location.search.substring(1));
 const gameID = params.get("gameID");
 
-document.addEventListener('DOMContentLoaded', function() {
+onConfigLoad(function() {
```

Comment:
I did not understand this part before either. What does it do?

Reply:
This executes the function passed to it when config is done loading (the function is defined in frontend/config/config.js), to avoid referencing the configuration before it's loaded.